docs/changes/cosmology/14980.api.rst

```diff
@@ -0,0 +1,2 @@
+The private module `astropy.cosmology.utils` has been deprecated
```

Comment:
```suggestion
The private module ``astropy.cosmology.utils`` has been deprecated
```

Reply:
I though we weren't mean to make links in the changelog, as the links can break when referring to old coe.

Reply:
Correct. Double backticks means "don't crossref link".

Reply:
Lol. I saw your change as removing back ticks. My bad. Yes. I'll add them.